applications/upgradeability-by-proxy.md

```diff
@@ -0,0 +1,141 @@
+# Open Grant Proposal
+
+* **Project Name:** Smart Contract Upgradeability
+* **Team Name:** Fractal?
+* **Payment Address:** lolcrypto
+
+*The above combination of your GitHub account submitting the application and payment address will be your unique identifier during the program. Please keep them safe.*
+
+^ wat
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+Smart contract upgradeability is important, because bugs, and allowing for new
+features. Upgrades may be constrained, like needing to have votes to upgrade,
+or something. Very flexible, this proposal.
+
+It would be nice if it were easy to use this when building with the smart
+contracts pallet.
+
+Our team is interested in this because something.
```

Comment:
It is very likely that at least part of our development efforts will relate to smart contracts. Upgradeability is important not just because bugs but because product discovery will constantly provide new inputs.